composer.json

```diff
@@ -33,8 +33,8 @@
         "vimeo/psalm": "^3.12"
     },
     "scripts": {
-        "lint": "./vendor/bin/phpcs --ignore=tests/MockAPNSServer",
-        "fix": "./vendor/bin/phpcbf --ignore=tests/MockAPNSServer",
+        "lint": "./vendor/bin/phpcs -s --ignore=tests/MockAPNSServer/node_modules/",
```

Comment:
I wish this option had a `--` version, like `--show-sniffs`. I checked the help and it doesn't 😞  It's mysterious like this... 🤷